README.md

```diff
@@ -1,2 +1,72 @@
+
+## Built With
+
+- HTML and CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [OPTIONAL ] 
- [ ] This live demo link is not working . Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users